frontend/javascripts/oxalis/view/action-bar/create_animation_modal.tsx

```diff
@@ -167,7 +168,14 @@ function CreateAnimationModal(props: Props) {
         `You selected too many meshes for the animation. Please keep the number of meshes below ${MAX_MESHES_PER_ANIMATION} to create an animation.`,
       );
 
-    const validationStatus = hasEnoughMags && isDtypeSupported && isDataset3D && !isTooManyMeshes;
+    const isBoundingBoxEmpty = boundingBox.getVolume() === 0;
```

Comment:
But here the BBox is not intersected with the color layer.

-> a color layer of bbox (0,0,0,10,10,10) and a bbox of (10,10,10,10,10,10) would also lead to zero volume. But this is not properly checked here.

Reply:
Or does such a scenario not result in an error?
I would at least say, that this would be an unwanted scenario, no part of the color layer would be rendered